### DISCUSSION TITLE
Support RethinkDB

### DISCUSSION THREAD

**Discussion by Yellowcarboy** [tiredofit/docker-db-backup#184] [Ideas]
Hello

A great addition would be to add the support rethinkdb. https://rethinkdb.com/docs/backup/
It appears to work well with the existing scripts. If you want to support it I can help do the development. 

Cheers,

Jonathan

**Comment by tiredofit** [comment]
Ahh - This image once supported it before it hit `2.0.0` here: https://github.com/tiredofit/docker-db-backup/blob/955a08a21b97748f845ebef29d85d1c2f7ad4242/install/etc/services.available/10-db-backup/run#L201 

I removed it due to development stopping. It seems that development has now restarted so it certainly is a candidate to be added in - I'm focussing on trying to get some client work right now so when thats successful I can commit some of my time to readding.

**Reply by Yellowcarboy** [reply]
Let me know if you need/want help.
